### ISSUE TITLE
Give a default value for some options

### ISSUE THREAD

**Issue by jttuboi** [widgetbook/widgetbook#171]
**Description**
Every time I save file of the Widgetbook, the flutter web force to draw all app again, so the problem is:

- the right side is always reloaded opened
- or the zoom is always reloaded 1x

**Expected Behavior**
Can set default values on code to always restart with my options, for example, instead start with zoom 1x, I set to start with 0.75x.

One recommendation is:
```dart
appInfo: AppInfo(name: 'Book', zoom: 0.75, rightSideClosed: true, ...);
```

**Comment by JensHor** [comment]
Thanks for raising this feature request 👍🏻
While you're waiting for this to be implemented let me explain whats happening. Flutter web currently does not support hot reload and instead hot restarts the application (see https://github.com/flutter/flutter/issues/53041). Therefore, the app state is reseted as you've described. 

If you need a workaround, we recommend to use [Flutter Desktop](https://docs.flutter.dev/desktop) because it feels the same as the web app, but has hot reload support.

**Comment by hmbenhaim** [comment]
> Thanks for raising this feature request 👍🏻 While you're waiting for this to be implemented let me explain whats happening. Flutter web currently does not support hot reload and instead hot restarts the application (see [flutter/flutter#53041](https://github.com/flutter/flutter/issues/53041)). Therefore, the app state is reseted as you've described.
> 
> If you need a workaround, we recommend to use [Flutter Desktop](https://docs.flutter.dev/desktop) because it feels the same as the web app, but has hot reload support.

I wish to do so, but some of the plugin I use dose not support desktop, only mobile and web. and I developed a bit on mobile but you can't get anything done only if you want to check functions, about design and layout have to use something with big screen, now the desktop reach stable I hope more plugins will be available and I can do so, or the web will get het reload.

**Comment by jenshor** [comment]
I'm closing this issue since we have introduced the following features in the beta release `widgetbook 3.0.0-beta.x`:
- deep linking to allow hot restarting of Widgetbook on Web
- addon implementation which allows to specify default values for theming, text-scales, frames and locales. 

We are currently moving quickly towards a stable Widgetbook 3 release so stay tuned!

**Comment by jenshor** [comment]
I'm closing this issue as I see it as resolved with the `widgetbook 3.0.0-beta.x` version. However, feel free to reopen if you have additional feedback 😊
